content/backstage/plugins/sonarqube.md

```diff
@@ -46,14 +46,13 @@ gettingStarted:
   - intro: Add the proxy config for SonarCloud
```

Comment:
I would point out that the reader can scroll down to the notes section to learn how to generate the `SONARCLOUD_AUTH` token.

Reply:
Hey Nic. I'm just realising now that the instructions do already point out that the reader can down to the notes section to learn how to generate the `SONARCLOUD_AUTH` token. It's just that the pointer is below the code samples rather than above. I totally misunderstood what it was telling me because of this. If you move the "Get and provide SONARQUBE_AUTH/SONARCLOUD_AUTH as env variables (see Notes on how to generate these values)" section above the code samples I think this will work better.

Reply:
Done!